src/Foundation/NSUrlSessionConfiguration.cs

```diff
@@ -64,5 +65,40 @@ public static NSUrlSessionConfiguration CreateBackgroundSessionConfiguration (st
 			return config;
 		}
 
+		/*
```

Comment:
I think you can delete this comment, ProxyConfigurations is bound in the api definitions.